src/Resque/Worker.php

```diff
@@ -760,9 +760,9 @@ public function queueDelayed($endTime = null, $startTime = 0)
             $this->redis->multi();
             $jobs = $this->redis->zrangebyscore(Queue::redisKey($queue, 'delayed'), $startTime, $endTime);
             $this->redis->zremrangebyscore(Queue::redisKey($queue, 'delayed'), $startTime, $endTime);
-            $res = $this->redis->exec();
-            if(!is_array($res) || count($res) != 2) return;
-            list($jobs, $found) = $res ;
+            $execReturn = $this->redis->exec();
+            if(!is_array($execReturn) || count($execReturn) != 2) return;
```

Comment:
According to PSR-2, this should be
```php
if (!is_array($execReturn) || count($execReturn) !== 2) {
    return;
}
```
Note: The comparision is changed to a type-safe one, as count() must return an integer according to the [docs](http://php.net/count).